code/modules/mob/living/simple_animal/friendly/drone/_drone.dm

```diff
@@ -80,7 +80,6 @@
 		equip_to_slot_or_del(I, slot_head)
 
 	access_card.flags |= NODROP
-	scanner.Grant(src)
```

Comment:
drones lose their scanner. (consistency with robots not having it, and also makes things easier codewise)